rust/backend/ebpf/src/sys_sendmsg.rs

```diff
@@ -23,13 +23,7 @@ struct SysSendmsgIntern {
 
 #[tracepoint]
 pub fn sys_enter_sendmsg(ctx: TracePointContext) -> u32 {
-    let pid = ctx.pid();
-
-    if unsafe { PIDS_TO_TRACK.get(&pid).is_none() } {
-        return 1;
-    }
-
-    let id = generate_id(pid, ctx.tgid());
```

Comment:
I think we still should check on `sys_enter` and only take the ones we are interested in, because:

- (very small edgecase: sys_enter happens at 1, enabling happens at 2, sys_exit happens at 3, how we get an event that happened before we enabled it)
- every single process will be inserted into the hashmap which increases the likelyhood of collisions a lot

Reply:
Thanks for the hint, I've implemented it in the latest commit